cpp/tensorrt_llm/kernels/userbuffers/CMakeLists.txt

```diff
@@ -14,7 +14,7 @@
 # License for the specific language governing permissions and limitations under
 # the License.
 #
-if(ENABLE_MULTI_DEVICE EQUAL 0)
+if(ENABLE_MULTI_DEVICE)
```

Comment:
This condition is inverted.

Reply:
Yes, tks, I ve fixed and repushed 30mn ago.

Reply:
oops, sorry, done.